mods/ra/rules/defaults.yaml

```diff
@@ -541,7 +541,6 @@
 	Inherits@bounty: ^GlobalBounty
 	Inherits@selection: ^SelectableCombatUnit
 	Inherits@handicaps: ^PlayerHandicaps
-	Huntable:
```

Comment:
We probably need to add this to `^Plane` then?

Reply:
Done.